GITOPS-CONFIG.md

```diff
@@ -0,0 +1,109 @@
+- [yq](https://github.com/mikefarah/yq/) version 4.0+.
+
+### AI-RHDH-Installer: Script Configuration
+If you installed using the `ai-rhdh-installer` all that is required for hooking up the ArgoCD instance to your RHDH instance is to run `bash ./scripts/configure-gitops.sh`. Please note if you changed the installation namespace used by the installer you will first need to run `export NAMESPACE=<namespace used>` as the default value is `ai-rhdh`.
+
+### AI-RHDH-Installer: Manual Configuration
+
+#### Step 1: ConfigMaps and Secrets
+In your chosen namespace you should apply the following ConfigMaps:
+1.  [`argocd-config.yaml` ](./resources/argocd-config.yaml)
+2.  [`argocd-plugins.yaml`](./resources/argocd-plugins.yaml)
+
+As part of the `ai-rhdh-installer` a secret was created in your desired namespace with the name `rhdh-argocd-secret`, keep note of this Secret as well as the two ConfigMaps applied above.
+
+#### Step 2: Updating Plugins
+**Note: RHDH will encounter errors if the ArgoCD plugins are loaded before an instance is properly attached - This step may result in errors until all steps are completed**
+
+To include the ArgoCD plugins we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
+
+![Dynamic Plugins Example](./assets/dynamic-plugins-example.png)
+
+Edit the associated `yaml` file to include `argocd-plugins.yaml` under the `includes` section:
+
+![Dynamic Plugins Example 2](./assets/dynamic-plugins-example-2.png)
+
+#### Step 3: Updating RHDH Deployment
+Now that all of the required ConfigMaps and Secrets are apart of the namespace, as well as adding the plugins we must edit the RHDH Deployment to properly reference these items.
+
+Since we are referencing `argocd-plugins.yaml` in our dynamic-plugins ConfigMap we first must add `argocd-plugins.yaml` as a `volumeMount` to the `initContainers` field in the RHDH Deployment under `.spec.template.spec.initContainers.volumeMounts`
+
+![ArgoCD Plugins Addition](./assets/argocd-plugins-example.png)
+
+Next we will need to add the `argocd-config.yaml` as a `VolumeMount` to the `containers` field in the RHDH Deployment under `.spec.template.spec.containers.volumeMounts`
+
+![ArgoCD Config Addition](./assets/argocd-config-example.png)
+
+Next add the created `rhdh-argocd-secret` to the `envFrom` field in the RHDH Deployment under `.spec.template.spec.containers.envFrom`
+
+![ArgoCD Secret Addition](./assets/argocd-secrets-example.png)
+
+Now we must update the config args used by RHDH to include our ArgoCD ConfigMap. This is added to the `args` field in the RHDH Deployment under `.spec.template.spec.containers.args`
+
+![ArgoCD Config Update](./assets/argocd-config-addition.png)
+
+Finally we need to add the resources included as `volumeMounts` to the `volumes` field in the RHDH Deployment under `.spec.template.spec.volumes`
+
+![ArgoCD Volumes Addition](./assets/argocd-volumes-example.png)
+
+### Pre-Existing Instance: Script Configuration
```

Comment:
Would it be better here if we explicitly mention which re the pre-existing instances?

Reply:
Are you referring to which instances should be pre-existing? I think there is a note at the start of the section stating you need RHDH pre existing and setup and then the section is all under gitops, but I can rename it to be more clear if that's what you are talking about?